src/game/behaviors/hoot.inc.c

```diff
@@ -194,7 +194,8 @@ void hoot_action_loop(void) {
 
 void hoot_turn_to_home(void) {
     s16 pitchToHome, yawToHome;
-    vec3f_get_angle(&o->oPosVec, &o->oHomeVec, &pitchToHome, &yawToHome);
+    f32 distToHome;
```

Comment:
Even though this is set with a macro, does it throw an unused variable warning or no?

Reply:
nope there are no warnings